papermill/exceptions.py

```diff
@@ -21,8 +23,11 @@ def __init__(self, exec_count, source, ename, evalue, traceback):
         self.traceback = traceback
         message = "\n" + 75 * "-" + "\n"
         message += 'Exception encountered at "In [%s]":\n' % str(exec_count)
+        tb = ""
         for line in traceback:
-            message += line + "\n"
+            tb += line + "\n"
+        ansi_escape = re.compile(r'\x1b[^m]*m')
```

Comment:
The complete regex you probably want is `r'\x1b\[[\d;]+m`. However, there is a module called `ansiwrap` with a function `strip_color`.